### ISSUE TITLE
Fix visual bugs on home

### ISSUE THREAD

**Issue by Torgian** [consento-org/consento-website#3]
- [x] Safari doesn't load main.css, but Chrome does.
- [x] Missing image. Change it to match design.
- [x] Minor visual bugs.

**Comment by martinheidegger** [comment]
This refers to PR #1

**Comment by Torgian** [comment]
Minor visual bugs are in mobile view for the most part. Button spacing and some text spacing.

**Comment by Torgian** [comment]
Finished up fixing a bunch of visual bugs ( especially in mobile view ).

As of this comment: I have made mobile view remove a bunch of spacing that only looks good on desktop mode. Media query is currently set to remove desktop spacing up to 500 pixels in width. This can be changed easily as we test. 

Right now, I'm trying to figure out why Safari won't load main.css, but I am unable to reproduce this bug locally. Will continue testing.

**Comment by Torgian** [comment]
Fixed the css mapping bug. Apparently this is a bug with Ghost's main.css file.

At the end of the file, we have this line:

```
/*# sourceMappingURL=main.css.map */
```

While Chrome sees this as a comment, Safari does not. Removing the `#` will tell Safari that this is a comment. Removing this hash does not affect Chrome either, so I have removed this #. I have also removed it from the bootstrap.min.css and .js files

As far as I can tell, Ghost hasn't included a .map file for its css, and this is a bug that other Ghost developers have encountered.
